docs/section11/Rule11-14.md

```diff
@@ -0,0 +1,37 @@
+# Service_Water_Heating - Rule 11-14  
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** FALSE  
+
+**Rule ID:** 11-14  
+
+**Rule Description:** "Where recirculation pumps are used to ensure prompt availability of service water-heating at the end use, the energy consumption of such pumps shall be calculated explicitly."  
+
+**Rule Assertion:** UNDETERMINED / NOT_APPLICABLE
+
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, (f)
+
+**Evaluation Context:** B_RMD - each building
+**Data Lookup:**   
+**Function Call:** 
+- NONE
+
+**Applicability Checks:**
+- check that there is SWH in the proposed model AND that the building has a 24hr operating schedule
+
+## Applicability Checks:
+- look at each building in the RMD: `for building in B_RMD:`
+    - create a boolean is_applicable and set it to false: `is_applicable = false`
+    - look at each space in the building: `for space in P_RMD.building...:`
+        - check to see if there is at least one SWH use associated with the space: `if len(space.service_water_heating_uses) > 0:`
+            - the rule is applicable: `is_applicable = true`
+      
+    
+    ## Rule Assertion:
+    - if the boolean is_applicable is equal to true, the rule is applicable, return UNDETERMINED and a note letting the reviewer know what to check: `if is_applicable: UNDETERMINED raise_message: "This building has service water heating loads.  Where recirculation pumps are used to ensure prompt availability of service water-heating at the end use, the energy consumption of such pumps shall be calculated explicitly."`
```

Comment:
Is there no where in the schema that captures recirulation pumps? ServiceWaterHeatingConfigurationOption
s has options that include recirculation. Perhaps we should add pump entries to the schema in order to assess this rule?

Reply:
In the schema, there can be pumps connected to the service water heating distribution piping.  However, this could be a well pump, or booster pump, not just a recirculation pump.

The language of this rule is interesting because it seems to be reminding people to include recirculation pumps (they're often forgotten) - this could be a proposed / user model comparison.  

There's also the `ServiceWaterPiping.is_recirculation_loop` which could be used

Let's discuss on call